phy/io/kwik/model.py

```diff
@@ -1136,9 +1193,7 @@ def n_clusters(self):
     # -------------------------------------------------------------------------
 
     def close(self):
-        """Close the `.kwik`, `.kwx`, and `.raw.kwd` files if they are open."""
+        """Close the `.kwik` and `.kwx` files if they are open."""
         if self._kwx is not None:
             self._kwx.close()
-        if self._kwd is not None:
-            self._kwd.close()
```

Comment:
hrmmmmmmmmmm. I guess you're right, they should be closed. and the .dat as well...